helm/dex-app/templates/_helpers.tpl

```diff
@@ -81,3 +81,18 @@ Abstract the knowledge to know if it needs a Giant Swarm connector or not
   {{- printf "false" }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Abstract the knowledge to know if its installed on a workload cluster or not
+*/}}
+{{- define "is-workload-cluster" -}}
+{{- if .Values.isWorkloadCluster -}}
```

Comment:
We check for WC first, so if both MC and WC are true, it will be considered a WC. The reason is that checks are in that order (with `else if`) within the helm chart as well, so behavior shouldn't change.